activities/Markdown.activity/locale.ini

```diff
@@ -0,0 +1,23 @@
+[*]
+sample-input=Sample input
+need-help=Do you need help?
+insertText=insertText
```

Comment:
Please add also text for others buttons: "wmd-bold-button-second", "wmd-italic-button-second", "wmd-heading-button", "wmd-hr-button", "wmd-olist-button", "wmd-ulist-button", "wmd-code-button", "wmd-quote-button", "wmd-link-button", "wmd-undo-button", "wmd-redo-button"